exotic/exotic.py

```diff
@@ -1258,24 +1256,24 @@ def fit_centroid(data, pos, init=[], box=10, debug=False):
             box=box  # only fit a subregion +/- 5 px from centroid
         )
     except:
-        log.info(f"WARNING: trouble fitting Gaussian PSF to star at {wx},{wy}")
+        log.info(f"WARNING: trouble fitting Gaussian PSF to star at {wfx},{wfy}")
         log.info("  check location of comparison star in the first few images")
         log.info("  fitting parameters are out of bounds")
         log.info(f"  init: {init}")
-        log.info(" lower:", [wx-5, wy-5, 0, 0, 0, -np.pi/4, np.nanmin(data)-1])
-        log.info(" upper:", [wx+5, wy+5, 1e7, 20, 20, np.pi/4, np.nanmax(data[yv, xv])+1])
+        log.info(" lower:", [wfx - 5, wfy - 5, 0, 0, 0, -np.pi/4, np.nanmin(data) - 1])
+        log.info(" upper:", [wfx + 5, wfy + 5, 1e7, 20, 20, np.pi/4, np.nanmax(data[yv, xv]) + 1])
 
         # use LM in unbounded optimization
         pars = fit_psf(
-            data, [wx, wy], init,
-            [wx-5, wy-5, 0, 0, 0, -PI/4, np.nanmin(data)-1],
-            [wx+5, wy+5, 1e7, 20, 20, PI/4, np.nanmax(data[yv, xv])+1],
+            data, [wfx, wfy], init,
+            [wfx - 5, wfy - 5, 0, 0, 0, -PI/4, np.nanmin(data) - 1],
+            [wfx + 5, wfy + 5, 1e7, 20, 20, PI/4, np.nanmax(data[yv, xv]) + 1],
```

Comment:
changes wx-> wfx and wy->wfy

Reply:
Yes, good call. We will want to use the weighted flux estimate